### DISCUSSION TITLE
Paypal BETA

### DISCUSSION THREAD

**Discussion by haidernaqvi87** [flutter-stripe/flutter_stripe#1111] [Q&A]
Do you guys plan to include full support of paypal intergeration provided the user has already configured paypal beta on stripe. I saw paypal was include in version 3.0 but then no mention of it. Any thorough example how to implement paypal using flutter stripe.

**Comment by remonh87** [comment]
Hi, see https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/regional_payment_methods/paypal_screen.dart
